api-core/src/main/java/com/vimeo/networking2/internal/VimeoCall.kt

```diff
@@ -50,6 +51,11 @@ interface VimeoCall<T> {
      */
     fun cancel()
 
+    /**
+     * The [HttpUrl] to which this call will be made.
+     */
+    val url: HttpUrl
```

Comment:
Perhaps this should be a `String` instead so that there is looser coupling to okhttp. `VimeoCall` is an internal class though, so it doesn't feel like violating the contract we have of not exposing implementation details.

Reply:
I'd vote for making this a `String`. 

All the current instances of it being called immediately convert it to a `String`, so unless you foresee a reason down the line where we'd need the URL as a `HttpUrl` instead of a `String` I see no reason for it to not be a `String`.